landlab/values/synthetic.py

```diff
@@ -333,7 +333,7 @@ def _get_x_and_y(grid, at):
     return x, y
 
 
-def constant(grid, name, at="node", where=None, value=0.0):
+def constant(grid, name, at="node", where=None, value=0.0, dtype=None):
```

Comment:
I think this is great.

Reply:
Cool. Note that `constant(grid, "x", value=10)` will create a field of integers - not floats. Since this follows the same behavior as numpy functions like `arange` I figured it was fine.